CMakeLists.txt

```diff
@@ -228,6 +228,15 @@ if ( NOT ${WRF_CASE} IN_LIST WRF_CASE_OPTIONS )
   message( FATAL_ERROR "WRF Case option not recognized : ${WRF_CASE}" )
 endif()
 
+# handle Hydro options
```

Comment:
Placement of this logic is best suited around line `254` after string selection options are handled

Reply:
Moved this section further down, now at line `265`